client/src/app/site/pages/meetings/pages/participants/pages/structure-levels/components/structure-level-list/structure-level-list.component.ts

```diff
@@ -58,6 +58,7 @@ export class StructureLevelListComponent extends BaseMeetingListViewComponent<Vi
         super.setTitle(`Structure Levels`);
         this.structureLevelForm = this.formBuilder.group({
             name: [``, Validators.required],
+            color_check: false,
```

Comment:
Shouldn't this be `[false]`?

Reply:
There is no difference in `false` or `[false]` it seems. It work both.